it/helpers/WireMockMethods.scala

```diff
@@ -81,31 +67,11 @@ trait WireMockMethods {
     }
   }
 
-  def verify(method: HTTPMethod, uri: String): Unit = verifyInternal(method, uri, None)
-  def verify[T](method: HTTPMethod, uri: String, body: T)(implicit writes: Writes[T]): Unit = {
-    val stringBody = writes.writes(body).toString()
-    verifyInternal(method, uri, Some(stringBody))
-  }
-
-  private def verifyInternal(method: HTTPMethod, uri: String, bodyString: Option[String]): Unit = method match {
-    case GET => WireMockHelper.verifyGet(uri)
```

Comment:
Can you review `WireMockHelper` and remove unused methods and vals please?
i.e. `stubPost()`, `verifyGet()`, `verifyPost()`, `wsClient`